gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/propertytransformation/PropertyTransformationTest.java

```diff
@@ -0,0 +1,365 @@
+      "g12:A { a : 42L } [(:A { a : 42000L, b : 42 })-[:a { a : 42000L, b : 42 }]" +
+      "->(:B { a: 42000L, c : 23 })]" +
+      "g22:A { dividedA : 42L, a : 42000L } [(:A { a : 42000L, b : 42 })-[:a { a : 42000L, b : 42 }]" +
+      "->(:B { a: 42000L, c : 23 })]" +
+      // vertex only transformation
+      "g03:A { a : 42000L } [(:A { a : 42L, a__1 : 42000L, b : 42 })-[:a { a : 42000L, b : 42 }]" +
+      "->(:B { a: 42L, a__1 : 42000L, c : 23 })]" +
+      "g13:A { a : 42000L } [(:A { a : 42L, b : 42 })-[:a { a : 42000L, b : 42 }]" +
+      "->(:B { a: 42L, c : 23 })]" +
+      "g23:A { a : 42000L } [(:A { dividedA : 42L, a : 42000L, b : 42 })" +
+      "-[:a { a : 42000L, b : 42 }]->(:B { dividedA: 42L, a : 42000L, c : 23 })]" +
+      "g33:A { a : 42000L } [(:A { a : 42L, a__1 : 42000L, b : 42 })-[:a { a : 42000L, b : 42 }]" +
+      "->(:B { a : 42000L, c : 23 })]" +
+      // edge only transformation
+      "g04:A { a : 42000L } [(:A { a : 42000L, b : 42 })-[:a { a : 42L, a__1 : 42000L, b : 42 }]" +
+      "->(:B { a: 42000L, c : 23 })]" +
+      "g14:A { a : 42000L } [(:A { a : 42000L, b : 42 })-[:a { a : 42L, b : 42 }]" +
+      "->(:B { a: 42000L, c : 23 })]" +
+      "g24:A { a : 42000L }  [(:A { a : 42000L, b : 42 })" +
+      "-[:a { dividedA : 42L, a : 42000L, b : 42 }]->(:B { a: 42000L, c : 23 })]" +
+      "g34:B  { a : 42000L } [(:A { a : 42000L, b : 42 })" +
+      "-[:a { a : 42L, a__1 : 42000L, b : 42 }]->(:B { a: 42000L, c : 23 })" +
+      "(:C { a : 42000L, b : 42 })-[:b { a : 42000L, b : 42 }]" +
+      "->(:D { a: 42000L, c : 23 })]";
+  /**
+   * Simple property transformation that divides a value by 1000.
+   */
+  protected static PropertyTransformationFunction DIVISION = new DivideBy(1000L);
+  /**
+   * Executes a property transformation on graphHeads, vertices and edges and checks if the result
+   * is correct.
+   *
+   * @throws Exception If the execution fails.
+   */
+  @Test
+  public void testAllTransformationFunctions() throws Exception {
+    FlinkAsciiGraphLoader loader = getLoaderFromString(TEST_GRAPH);
+    
+    LogicalGraph original = loader.getLogicalGraphByVariable("g0");
+
+    LogicalGraph expected = loader.getLogicalGraphByVariable("g01");
+
+    LogicalGraph result = original.callForGraph(new PropertyTransformation("a",
+    		DIVISION, DIVISION, DIVISION, null, null, true));
+
+    collectAndAssertTrue(result.equalsByData(expected));
+  }
+
+  /**
+   * Executes a property transformation on graphHeads and checks if the result is correct.
+   *
+   * @throws Exception If the execution fails.
+   */
+  @Test
+  public void testGHTransformation() throws Exception {
+    FlinkAsciiGraphLoader loader = getLoaderFromString(TEST_GRAPH);
+    
+    LogicalGraph original = loader.getLogicalGraphByVariable("g0");
+
+    LogicalGraph expected = loader.getLogicalGraphByVariable("g02");
+
+    LogicalGraph result = original.callForGraph(new PropertyTransformation("a", DIVISION, null,
```

Comment:
Since we are planing to increase the code style checks for test classes i prefer to fix to long lines.  Fix for all tests.

```
    LogicalGraph result = original.callForGraph(
      new PropertyTransformation(
        "a",
        DIVISION,
        null,
        null,
        null,
        null,
        true));
```